torch/csrc/Exceptions.h

```diff
@@ -271,8 +271,8 @@ TORCH_PYTHON_API std::string processErrorMsg(std::string str);
 
 // Abstract base class for exceptions which translate to specific Python types
 struct PyTorchError : public std::exception {
-  // NOLINTNEXTLINE(modernize-pass-by-value)
-  PyTorchError(const std::string& msg_ = std::string()) : msg(msg_) {}
+  PyTorchError() = default;
+  PyTorchError(std::string msg_) : msg(std::move(msg_)) {}
```

Comment:
This one is an actual functional change, are we sure we want to change this one? They might a good reason for doing this like rely on small string optimization?

Reply:
I think splitting the  constructor into two more specific ones give the compiler more chance to optimisation. I can't see any relation of that constructor with SSO..